README.md

```diff
@@ -6,12 +6,16 @@ Take a peek into your Rails application.
 
 ![Preview](https://f.cloud.github.com/assets/79995/244991/03cee1fa-8a74-11e2-8e33-283cf1298a60.png)
 
-This is a profiling tool originally built at GitHub to help us get an insight into our application. Now, we have extracted this into Peek, so that other Rails application can experience the same benefit.   
+No explanation of Peek is complete without discussing what Rails is. Rails is a framework written in the Ruby language for use in developing web applications. 
+
+Peek is a profiling tool originally built at GitHub to help us get an insight into our application. Now, we have extracted this into Peek, so that other Rails application can experience the same benefit.   
 
 Peek puts a little bar on top of your application to show you all sorts of helpful information about your application. From the screenshot above, you can see that Peek provides information about database queries, cache, Resque workers and more. However, this is only part of Peek's beauty.
 
 The true beauty of Peek lies in the fact that it is an extensible platform. If there are some performance metrics that you need but are not available on Peek, you can find it from the list of available [Peek Views](#available-peek-views) and integrate it into Peek. Even if you do not find what you want on Peek Views, you can always [create your own](#creating-your-own-peek-item).
 
+A Peek Item is simply a class that will fetch and report the data to be displayed on the Peek bar. In order to do this, Peek keeps track of every request made by the Rails application you are building. By default, this information is stored in memory. To use Peek, it must be loaded into the environment. It can be enabled to see what’s going on by looking at the stats on the Peek bar. Peek has the option to display only the required info, so that your bar won’t get crowded. 
```

Comment:
I feel like this is just a rehash of what is said at https://github.com/peek/peek/pull/87/files#diff-04c6e90faac2675aa89e2176d2eec7d8R55 and https://github.com/peek/peek/pull/87/files#diff-04c6e90faac2675aa89e2176d2eec7d8R79. Are you able to point out what is different and how they differ?